### PR TITLE
generlize redis options

### PR DESCRIPTION
**Pull request by ananthakumaran**
This is a continuation of #345 

* redis_worker option is no longer supported
* we no longer support < 1.6 (this is because redix doesn't support them any longer)

Need to do
* cleanup the readme about sentinel
* the flaky connection tests seem to fail locally, I haven't debugged it further.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-elixir:
-  - 1.3.2
-  - 1.4.0
-otp_release:
-  - 18.0
-  - 19.0
+matrix:
+  include:
+    - language: elixir
+      otp_release: 20.3
```
Comment (akira) [inline .travis.yml]: Just to check, are we removing support for earlier than Elixir 1.6?

Comment (ananthakumaran) [reply .travis.yml]: Yes, redix dropped support for in 0.8 https://github.com/whatyouhide/redix/blob/master/CHANGELOG.md#v080

Comment (akira) [reply .travis.yml]: 👍OK, sounds good, I can approve PR.  Do you have a chance to resolve conflicts on this? I merged a cleanup PR and appears it introduced conflicts with this.

Comment (ananthakumaran) [reply .travis.yml]: sure, will fix the conflict

**Comment by ananthakumaran** [comment]
~* cleanup the readme about sentinel~
~* the flaky connection tests seem to fail locally, I haven't debugged it further.~

fixed both of the issues

**Comment by akira** [comment]
👍
@ananthakumaran  Thank you for these fixes!!  👏 👏

**Comment by gergo-papp** [comment]
Thanks @ananthakumaran
@akira will you release a new version to hex with this?

Happy holidays!

**Comment by nyo16** [comment]
Thank for the changes guys! 
Any plans for new version in hex?

**Pull request merged**